src/test/kotlin/com/seokjoo/todo/domain/service/charge/TodoChargeServiceTest.kt

```diff
@@ -0,0 +1,28 @@
+package com.seokjoo.todo.domain.service.charge
+
+import com.seokjoo.todo.domain.service.auth.TodoAuthService
+import org.junit.jupiter.api.Test
+import org.springframework.beans.factory.annotation.Autowired
+import org.springframework.boot.test.context.SpringBootTest
+import org.springframework.test.context.ActiveProfiles
+import org.springframework.transaction.annotation.Transactional
+
+@SpringBootTest
+@ActiveProfiles("test")
```

Comment:
```suggestion
@IntegrationTest
```

만들어두신 어노테이션 사용하면 될 것 같으네요 ?!

Reply:
바보같이 이전에 만들어둔 어노테이션안에 @Transactional을 붙여놨던 것 같은데 ,,, 빼고 붙일까용

Reply:
엇 아니엿네용 변경 완료 a5264811a5b2d445ac475e2ae4927e2dc019b99a

Reply:
아니넹 저번에 뺏네요 ㅋㅅㅋ;